### PR TITLE
Add Differentially Private Tabular Data Synthesis using LLMs

### PR DESCRIPTION
**Pull request by toan-vt**

### CODE REVIEW

**Comment by personalization-trap** [comment]
I think the paper is relevant. Can you write a short description of it so that it makes it easier for us to write it in the next version? Also, which section of survey should I include the paper?

**Comment by toan-vt** [comment]
I think it should be in the data generation part (Section 4)

Below is the short description (about methodology only -- as the current outline of the section)
DP-LLMTGen is the first to leverage pretrained LLMs for the tabular data generation problem with differential privacy constraints. GReaT can fail at format compliance in the differentially private settings due to noise added to gradients during training. To address this challenge, DP-LLMTGen employs a two-stage fine tuning mechanism that distinguishes learning targets (format compliance and data modeling) and treats them appropriately. Additionally, DP-LLMTGen implements a weighted cross-entropy and numerical-understanding loss function. This function aligns with the proposed fine-tuning procedure and improves LLMs' numerical understanding.
